master/internal/stream/system.go

```diff
@@ -106,22 +99,35 @@ func NewPublisherSet() *PublisherSet {
 	}
 }
 
+// SyncMsg is the server response to a StartupMsg once it's been handled.
+type SyncMsg struct {
+	SyncID string `json:"sync_id"`
+}
+
+// toPreparedMessage converts as SyncMsg to a PreparedMessage.
+func (sm SyncMsg) toPreparedMessage() *websocket.PreparedMessage {
```

Comment:
would this cause issues for testing since it's returning a preparedMessage?

Reply:
it could probably be hacked around, but yeah this isn't ideal if it's not analgous to the typing used for the other kinds of messages.

Reply:
After talking with Eric, the expectation should be to make this compatible with prepareFn() that way during testing we can actually see the value stored in the sync